src/config.h

```diff
@@ -61,6 +64,25 @@ typedef std::variant<ConfigKeyBool,
                      ConfigKeyUserSymbolCacheType>
     ConfigKey;
 
+// These strings match the env variables (minus the 'BPFTRACE_' prefix)
+const std::map<std::string, ConfigKey> CONFIG_KEY_MAP = {
+  { "node_max", ConfigKeyInt::ast_max_nodes },
```

Comment:
Similar to "verify_llvm_ir" - this one's just used for fuzzing: https://github.com/iovisor/bpftrace/pull/1601/commits/fb87a9f9f476af3d6cd6d3b42bea8bbd7235cc07
I'd say exclude.

>    This introduces `BPFTRACE_NODE_MAX` environment variable, and it
>    controls the maximum number of AST nodes. This feature is intended for
>    developers, and its usage is not included in the reference guide.